lib/infrastructure/repositories/transaction_builder.dart

```diff
@@ -94,3 +95,50 @@ extension TransferTransactionBuilder on archethic.Transaction {
         .originSign(originPrivateKey);
   }
 }
+
+// TODO(chralu): 2 extensions in a same file or 1 generic extension with all params from transaction object ?
```

Comment:
1 generic extension with all params.

Eventually 2 specific methods that use the generic one.